### PR TITLE
Added reverse proxy configuration example

### PR DESCRIPTION
**Pull request by manjerico**

### CODE REVIEW

**Comment by phanan** [comment]
Thanks, but isn't this outside of a `.htaccess`'s scope?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # .htaccess Snippets
-A collection of useful .htaccess snippets, all in one place. I decided to create this repo after getting so tired (and bored) with Googling everytime there's a need of forcing `www` for my new website.
```
Comment (gaomd) [inline README.md]: Seems valid to call them *snippets*, we should change all instance of `.htaccess snippets` to `.htaccess` if we really want to.

**Comment by manjerico** [comment]
You make a fair point, sir. My mistake, this is apache.conf stuff. I immediately assumed that's what it was, since the README.md source code included "``` apacheconf". Off with its head!

**Comment by phanan** [comment]
Closing as invalid.

**Pull request closed without merging**